src/fonts/OpenDyslexic/OpenDyslexic/OpenDyslexic-Bold.svg

```diff
@@ -0,0 +1,244 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd" >
```

Comment:
Hello @derekriemer in this commit there are duplicated font files (inside a new OpenDyslexic subfolder) in addition to src/js/ReaderSettingsDialog.js